client-admin/src/components/conversation-admin/index.js

```diff
@@ -114,6 +116,17 @@ const ConversationAdminContainer = () => {
             Report
           </Link>
         </Box>
+        {/* {zid_metadata.requires_invite_codes && ( */}
```

Comment:
uncomment to have this menu item only show up after a conversation is made to require invite codes